src/timezonepicker/types.ts

```diff
@@ -28,6 +28,9 @@ export type Timezone = {
    */
   offset: number;
 };
+export type TimezoneOverrides = {
```

Comment:
```suggestion
export type TimezonePickerOverrides = {
```

Reply:
that is what we have in old type definitions - https://github.com/uber/baseweb/blob/master/legacy-dts/timezonepicker/index.d.ts#L12-L14

Reply:
but yeah, looks like a good idea to rename it here and create deprecated alias with old name